packages/components/pie-textarea/test/visual/pie-textarea.spec.ts

```diff
@@ -142,3 +146,17 @@ test.describe('Resize mode:', () => {
         });
     });
 });
+
+test.describe('Label and Character count:', () => {
```

Comment:
Note for reviewers - the width will look a little funky for the textarea. This is a known issue that is resolved in a ticket being brought in next week 👍